inst/@sym/assumeAlso.m

```diff
@@ -30,15 +30,7 @@
 %% @group
 %% syms x integer
 %% x1 = x;
-%% assumptions(x1)
-%%   @result{} ans =
-%%     @{
-%%       [1,1] = x: integer
-%%     @}
```

Comment:
I guess its because I only added that test as a workaround because of the skipped test.

But I guess it does read ok with it, I'll put it back!